site/gatsby-config.js

```diff
@@ -28,13 +20,6 @@ const plugins = [
     options: manifestConfig,
   },
   'gatsby-plugin-styled-components',
-  {
-    resolve: 'gatsby-source-contentful',
```

Comment:
also uninstall it and update `package.json`

Reply:
Right, I used npm. My bad